pytermgui/ansi_interface.py

```diff
@@ -27,6 +28,13 @@
 _IS_NT = _name == "nt"
 _SYS_HAS_FRAME = hasattr(sys, "_getframe")
 
+if _IS_NT:
+    import ctypes
+
+    kernel32 = ctypes.windll.kernel32
+else:
+    import termios
+
```

Comment:
If OS is Windows import **ctypes** otherwise import **termios** (needed for mode changing)